src/uniprotkb/components/landing-page/LandingPage.tsx

```diff
@@ -0,0 +1,286 @@
+import { Fragment } from 'react';
+import { Link } from 'react-router-dom';
+import {
+  ExternalLink,
+  LongNumber,
+  SwissProtIcon,
+  TremblIcon,
+} from 'franklin-sites';
+import cn from 'classnames';
+
+import StatisticsChart from './StatisticsChart';
+
+import YouTubeEmbed from '../../../shared/components/YouTubeEmbed';
+
+import useDataApi from '../../../shared/hooks/useDataApi';
+import useUniProtDataVersion from '../../../shared/hooks/useUniProtDataVersion';
+
+import {
+  LocationToPath,
+  Location,
+  getLocationEntryPath,
+} from '../../../app/config/urls';
+import { getAPIQueryUrl } from '../../../shared/config/apiUrls';
+import ftpUrls from '../../../shared/config/ftpUrls';
+
+import { SearchResults } from '../../../shared/types/results';
+import { Namespace } from '../../../shared/types/namespaces';
+
+import styles from './styles/landing-page.module.scss';
+
+const availableFTPFormats = {
+  fasta: 'fasta',
+  text: 'dat',
+  xml: 'xml',
+};
+
+const tutorialsInfo = [
+  {
+    id: 'yp1O1gDK8oA',
+    title: 'How to search UniProtKB',
+    date: '26 Oct 2021',
+  },
+  {
+    id: 'BHu88Sv--mc',
+    title: 'How to explore a UniProt entry',
+    date: '17 Feb 2022',
+  },
+  {
+    id: 'p4_gGkM-Rfs',
+    title: 'How to download embeddings in UniProt',
+    date: '25 Jul 2023',
+  },
+];
+
+const LandingPage = () => {
+  const release = useUniProtDataVersion();
+
+  const { data } = useDataApi<SearchResults<never>>(
+    getAPIQueryUrl({
+      namespace: Namespace.uniprotkb,
+      query: '*',
+      size: 0,
+      facets: ['reviewed'],
+    })
+  );
+
+  let numberReviewed: undefined | number;
+  let numberUnreviewed: undefined | number;
+
+  if (data?.facets?.[0].values?.length) {
+    numberReviewed = data?.facets?.[0].values.find(
+      (value) => value.value === 'true'
+    )?.count;
+    numberUnreviewed = data?.facets?.[0].values.find(
+      (value) => value.value === 'false'
+    )?.count;
+  }
+
+  return (
+    <div className={styles['landing-page']}>
+      <h2 className={styles['landing-page__title']}>UniProtKB</h2>
```

Comment:
This should be a h1 (it's the title of the whole page), and the other headings after it should be h2, and if any heading within these subsections then it should follow and be h3.
If you need to play with the sizes because their default ones are either too big or too small check Franklin's typography file for the classes you can use (I think there are `.huge`, `.big`, etc)

Reply:
h3 looked very big for the subsections in statistics. I have used 'tiny'. Please have a look